Jenkinsfile

```diff
@@ -44,4 +44,29 @@ pipeline {
       }
     }
   }
+  post {
+    success {
+      script {
+        if (BRANCH_NAME == 'master' || TAG_NAME == 'production-release') {
```

Comment:
```
Error when executing success post condition:
groovy.lang.MissingPropertyException: No such property: TAG_NAME for class: groovy.lang.Binding
```

The Jenkins build is failing here.

Reply:
updated to use the `env.` prefix to access the ENV vars.